### PR TITLE
remove sha256 and rmd160 hash

### PR DESCRIPTION
**Pull request by winsvega**
connects to https://github.com/ethereum/solidity/issues/784

### CODE REVIEW

**Comment by bobsummerwill** [comment]
Right - so you hit the 10minute thing again here on Ubuntu, huh?
So yeah - I would just recommend that we insert some additional TTY generation into the long-running tests.

**Comment by chriseth** [comment]
@bobsummerwill there are no long-running tests in Solidity. This is caused by a bug in the mining routine in eth.

File: test/libsolidity/SolidityEndToEndTest.cpp [hunk 1]
```diff
@@ -1700,13 +1699,10 @@ BOOST_AUTO_TEST_CASE(sha256)
...
 							 "}\n";
 	compileAndRun(sourceCode);
-	auto f = [&](u256 const& _input) -> u256
-	{
-		return dev::sha256(dev::ref(toBigEndian(_input)));
-	};
-	testSolidityAgainstCpp("a(bytes32)", f, u256(4));
-	testSolidityAgainstCpp("a(bytes32)", f, u256(5));
-	testSolidityAgainstCpp("a(bytes32)", f, u256(-1));
+	std::cerr << toHex(u256(4));
```
Comment (chriseth) [inline test/libsolidity/SolidityEndToEndTest.cpp]: Please remove this.

File: test/libsolidity/SolidityEndToEndTest.cpp [hunk 2]
```diff
@@ -1700,13 +1699,10 @@ BOOST_AUTO_TEST_CASE(sha256)
...
+	testSolidityAgainstCpp("a(bytes32)", string("e38990d0c7fc009880a9c07c23842e886c6bbdc964ce6bdd5817ad357335ee6f"), u256(4));
```
Comment (chriseth) [inline test/libsolidity/SolidityEndToEndTest.cpp]: This should be a hex string, right? So you probably need `fromHex("e38...")`. Did you run these locally?

Comment (chriseth) [inline test/libsolidity/SolidityEndToEndTest.cpp]: Oh and I think using `testSolidityAgainstCpp` is not the right way here. You should use `BOOST_CHECK(callContractFunction(...) == ...);`

File: test/libsolidity/SolidityExecutionFramework.h [hunk 3]
```diff
@@ -128,6 +128,19 @@ class ExecutionFramework
 		);
 	}
 
+	template <class... Args>
+	void testSolidityAgainstCpp(std::string _sig, std::string const& _expectHash, Args const&... _arguments)
```
Comment (chriseth) [inline test/libsolidity/SolidityExecutionFramework.h]: Oh now I see. :-)
I don't think it is worth adding such a function here.

File: test/libsolidity/SolidityExecutionFramework.h [hunk 4]
```diff
@@ -128,6 +128,19 @@ class ExecutionFramework
 		);
 	}
 
+	/*template <class... Args>
```
Comment (chriseth) [inline test/libsolidity/SolidityExecutionFramework.h]: Please remove this.

**Comment by axic** [comment]
@winsvega @chriseth was it intentional to leave libdevcore/picsha2.h in the repo?

**Pull request merged**